Lykke.Service.LykkeService/client/Lykke.Service.LykkeService.Client/LykkeServiceClient.cs

```diff
@@ -1,15 +0,0 @@
-﻿using Lykke.HttpClientGenerator;
```

Comment:
why was the client deleted? in most cases you'll have several controllers and this client will aggregate them. Overwise you'll have to implement ILykkeServiceClient in one controller